internal/install/_static/Dockerfile.terraform_deployer

```diff
@@ -1,17 +1,14 @@
 FROM hashicorp/terraform:light as terraform
 
-FROM python:3-alpine
+FROM ubuntu:20.04
 
-# required by gcloud SDK
-RUN apk add --no-cache git openssh curl
+RUN apt-get -q update && apt-get install -yq curl apt-transport-https ca-certificates gnupg && apt-get clean
 
-ENV GCLOUD_SDK_VERSION 369.0.0
-ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
-RUN curl "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-$GCLOUD_SDK_VERSION-linux-x86_64.tar.gz" > /tmp/google-cloud-sdk.tar.gz \
-  && mkdir -p /usr/local/gcloud \
-  && tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
-  && /usr/local/gcloud/google-cloud-sdk/install.sh -q --override-components="bq" \
-  && rm /tmp/google-cloud-sdk.tar.gz
+ENV GCLOUD_SDK_VERSION 370.0.0-0
```

Comment:
Please move this to the top of the file (after FROM)

Reply:
done